spark/src/test/scala/io/delta/sharing/spark/DeltaSharingSourceSuite.scala

```diff
@@ -51,7 +51,7 @@ class DeltaSharingSourceSuite extends QueryTest
   lazy val toNotNullTable = testProfileFile.getCanonicalPath +
     "#share8.default.streaming_null_to_notnull"
 
-  lazy val deltaLog = RemoteDeltaLog(tablePath)
+  lazy val deltaLog = RemoteDeltaLog(tablePath, forStreaming = true)
```

Comment:
Is there a way to test that the request header has the streaming version?

Reply:
Added a test.